openfe/tests/setup/atom_mapping/test_perses_scorers.py

```diff
@@ -0,0 +1,67 @@
+# This code is part of OpenFE and is licensed under the MIT license.
+# For details, see https://github.com/OpenFreeEnergy/openfe
+
+import itertools
+import numpy as np
+from numpy.testing import assert_allclose, assert_
+import openfe
+from openfe.setup.atom_mapping import perses_scorers, LigandAtomMapping
+from perses.rjmc.atom_mapping import AtomMapper, AtomMapping
+
+atom_mapping_cls = openfe.setup.atom_mapping.LigandAtomMapping
+
+
+# full back to back test again lomap
+def test_perses_regression(lomap_basic_test_files_dir,  # in a dir for lomap
```

Comment:
I appreciate this is WIP, but this is quite a lot, is there a way to make this a lot cleaner by using parametrize & fixtures here?